Public/Sdk/SelfHost/MacOS/clang.dsc

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
exactly the same as before, just moved to parent directory and enclosed in `export namespace Clang`